prow/gerrit/adapter/adapter.go

```diff
@@ -183,6 +183,21 @@ func makeCloneURI(instance, project string) (*url.URL, error) {
 	return u, nil
 }
 
+// lists the files changed as part of a Gerrit patchset
+func listChangedFiles(changeInfo client.ChangeInfo) []string {
+	changed := make(map[string]bool)
```

Comment:
you probably want to use a `set` instead of `map[string]bool` (like https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/sets/string.go)